src/apps/concepts/components/ViewConceptsHeader.tsx

```diff
@@ -70,13 +70,25 @@ const ViewConceptsHeader: React.FC<Props> = ({
   gimmeAUrl,
   addConceptToDictionary,
   children,
-  sources,
+
 }) => {
   const [showSources, setShowSources] = useState(false);
-  const formatPrefferedSources = Object.entries(PREFERRED_SOURCES_VIEW_ONLY).map(([key, value]) => ({ name: key , url: value }));
+  const [preferredSources, setPreferredSources] = useState();
+useEffect(() => {
```

Comment:
You need to import `useEffect()` if your editor doesn't do this automatically, it goes in line 1 of this file, i.e.,

```ts
import React, { useEffect, useState } from "react";
```